src/util-print.c

```diff
@@ -292,3 +294,9 @@ const char *PrintInet(int af, const void *src, char *dst, socklen_t size)
     }
     return NULL;
 }
+
+void PrintHexString(char *str, size_t size, uint8_t *buf, size_t buf_len)
+{
+    DEBUG_VALIDATE_BUG_ON(size < 2 * buf_len);
+    rs_to_hex((uint8_t *)str, size, buf, buf_len);
```

Comment:
or alternatively we could do the nul termination here...

guess its also a bit sketchy that we use uint8_t as a c string

Reply:
Waiting for Jason to comment on this null termination...